webapp/docs/06_runtime_view.adoc

```diff
@@ -29,29 +29,87 @@ There are many notations for describing scenarios, e.g.
 * ...
 
 ****
+This section will have a contrast with the last, as we will cover here the dynamic behaviour and how users interact between them.
 
-=== <Runtime Scenario 1>
+=== Alice wants to know if Bob is near here, and Bob allows Alice to see him in the radar
 
 
-* _<insert runtime diagram or textual description of the scenario>_
-* _<insert description of the notable aspects of the interactions between the
-building block instances depicted in this diagram.>_
+* _Today, Alice is bored at home, but she wants to do something, taking into account COVID-19 restrictions.
+    As a consequence of these restrictions, she cannot leave her town, and wants to know if his best friend Bob
+    who lives in the same town as her, is near. As Bob allows Alice to find her in the application, Alice is able to access Bob's location in case he is near her._
+* _As you can see in the diagram below, at first Alice has to authenticate and allow sharing his location in order to find his mutuals between a proximity radius.
```

Comment:
and allow sharing her location

his -> her 
as we are talking about Alice not Bob